gui/freeadmin/forms.py

```diff
@@ -106,15 +107,18 @@ def _reroll(self):
             users = FreeNAS_Users(flags=FLAGS_DBINIT | FLAGS_CACHE_READ_USER)
         except:
             users = []
+        kwargs = {
```

Comment:
@william-gr Keyword arguments are created here for the fields and the exclude key is being added to them as a string. The resource json_user accepts kwargs and checks the exclude key either in get params or in kwargs and proceeds accordingly